sentinelhub/areas.py

```diff
@@ -533,7 +553,9 @@ def _get_utm_from_props(utm_dict):
         :return: UTM coordinate reference system
         :rtype: sentinelhub.CRS
         """
-        return CRS('32{}{}'.format(6 if utm_dict['direction'] == 'N' else 7, str(utm_dict['zone']).zfill(2)))
+        hemisphere_digit = 6 if utm_dict['direction'] == 'N' else 7
+        zone_digits = str(utm_dict['zone']).zfill(2)
+        return CRS(f'32{hemisphere_digit}{zone_digits}')
```

Comment:
This could be done via formatting `return CRS(f'32{hemisphere_digit}{utm_dict["zone"]:02d}')` but this might make it less readable

Reply:
Nice suggestion. Changed - I made a compromise between both options.